src/build.sh

```diff
@@ -247,9 +278,34 @@ function full_cleanup()
   cmd_manager "$flag" "$cmd"
 }
 
+# This functions uses iteractive 'git rebase' with '--exec' flag under the hood
+# to apply a 'kw build' over each commit from SHA to branch head.
+#
+# @flag How to display a command, see `src/lib/kwlib.sh` function `cmd_manager`.
+# @sha The SHA from the first commit to be compiled until the branch head.
+#
+# Return:
+# 0 if successfully compiled patchset, 1 otherwise.
+function build_from_sha()
+{
+  local flag="$1"
+  local sha="$2"
+  local cmd
+
+  flag=${flag:-'SILENT'}
+  cmd="git rebase ${sha} --exec 'kw build'"
+  cmd_manager "$flag" "$cmd"
+
+  if [[ "$?" != 0 ]]; then
+    echo "kw build failed during the compilation of a patch! Aborting rebase..."
+    git rebase --abort
```

Comment:
https://kworkflow.org/content/codingstyle.html#use-printf-instead-of-echo

If it fails in some specific commit, this is useful since the developer can try to fix the issue. Instead of just abort, ask the user if they want to abort.

Reply:
I've added a warning message reporting the failed rebase.